pkg/auto-pipeline.test.ts

```diff
@@ -34,7 +34,9 @@ describe("Auto pipeline", () => {
       redis.decrby(newKey(), 1),
       redis.del(newKey()),
       redis.echo("hello"),
+      redis.eval_ro("return ARGV[1]", [], ["Hello"]),
```

Comment:
`eval_ro` doesn't fit the js convention I think. ioredis doesn't have this method. In [node-redis they use evalRo](https://github.com/redis/node-redis/blob/d64072da95a46331a9bd3857c221c4f889d79259/packages/client/lib/commands/EVAL_RO.spec.ts#L21). I think evalRo would be better. What do you think?